proposed/project-file-simplification.md

```diff
@@ -0,0 +1,351 @@
+
+`Import` and `Base` will work well together and the `Base` attribute can appear in any `Import` file.
+
+#### Multiple `Base` files
+
+We considered whether to allow more than one `Base` files. We have three options:
+
+* Allow multiple `Base` files and evaluate in an order that maintains precedence
+* Throw an error if more than one `Base` is encountered during evaluation
+* Ignore all but the `Base` that is closest to the project file
+
+We do not have scenarios where multiple `Base` files all need to be evaluated. We do have scenarios where multiple `Base` declarations appear, but only the one closest to the project should be evaluated. This is just an extension of MSBuild principles about recognizing the properties closest to the user. The leading project will use the defaults of the new `TargetFramework` ignoring the earlier .NET version's `Base` file.
+
+Only the first `Base` encountered will be used.
+
+#### Immutability/AssignOnce
+
+We have received a request to allow some specific values to block further change to the values set from the `Base`. We do not see a reason for this in the `Base` files we will create.
+
+We are exploring broader use of the `AssignOnce` attribute on properties that make them immutable. We think this may solve the request to block further property changes in `Import` files.
+
+#### Abbreviating version declaration
+
+The SDK version number changes quarterly, while most breaking changes and warnings are introduced annually. Also, when users think about the version their behavior is tied to, users may think in terms of aligning to the major .NET version. Thus, supporting a short hand of just `10`, rather than `10.0.100` is desirable. If the user wants to use a quarterly `Base` file, they will need to enter the full SDK version number. We anticipate quarterly changes to `Base` files to be rare, except in support of upcoming preview features.
+
+### Changes to common user files
+
+We will add two new attributes to project files. This will be an optional/alternative way to declare the SDK and the property values that were previously in the project file when created from templates. The existing approach to project files will remain unchanged.
+
+In addition, we will provide a mechanism for declaring a default property value. When this file is read a check will be made - if the property has already been set, the existing value will be retained and no action will be performed.
+
+#### `Type` attribute
+
+_The name `Type` is not final._
+
+The `SDK` attribute on the `Project` element can be replaced by a `Type` attribute. It is not legal to have both a `Type` and an `SDK` attribute. If neither exists, the project will behave in the same manner as projects without a `SDK` attribute behave today.
```

Comment:
For Type to replace Sdk think we need some very strong motivating examples cause it will create dialects of the language.

Reply:
Do you think this is any more than other features?